src/components/ExchangeSummary.tsx

```diff
@@ -119,7 +119,7 @@ const ExchangeSummary = (props: {
   }
   return (
     <>
-      <TableData>{props.exchange}</TableData>
+      <TableData>{EXCHANGETOSTRING[props.exchange]}</TableData>
```

Comment:
There's now a merge conflict here. Can you fix this?

Reply:
Please see Notion Task on Wizardary Board for additional information issue I've encountered. Also messaged in Discord